core/src/main/java/com/amplifyframework/api/graphql/error/GraphQLLocation.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api.graphql.error;
```

Comment:
Since `GraphQLResponse` is at `com.amplifyframework.api.graphql`, and since `Error lives in it, that means Error is com.amplifyframework.api.graphql`. All of these inner components should live in the same package as well.

In other words, I think this package should be `com.amplifyframework.api.graphql`.

Reply:
Makes sense, I will move this as well as GraphQLPathSegment!